app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationActivity.java

```diff
@@ -106,7 +98,6 @@ protected void onCreate(Bundle savedInstanceState) {
                 .onPositive((@NonNull MaterialDialog dialog, @NonNull DialogAction which) -> {
                     int newTextSize = setTextViewHolder.textSizeSlider.getProgress() + 1;
                     preferencesManager.setPresentationTextSize(newTextSize);
-                    names.setTextSize(TypedValue.COMPLEX_UNIT_SP, newTextSize * 8);
```

Comment:
Users get mad when you disable random features. I think we should make the text size at the adapter layer dynamic and call notifyDataSetChanged() when this happens.